src/mpyl/cli/build.py

```diff
@@ -360,10 +404,34 @@ def ask_for_tag_input(ctx, _param, value) -> Optional[str]:
     is_flag=False,
     flag_value="prompt",
     default="not_set",
-    callback=ask_for_tag_input,
+    callback=ask_for_input,
+)
+@click.option(
+    "--version",
+    "-v",
+    is_flag=False,
+    flag_value="prompt",
+    default="not_set",
+    envvar="MPYL_RELEASE",
```

Comment:
I don't understand in what scenario you would want to set this version via an env var. the jenkins command never runs in a CI context.